project_sla/report/report_sla.py

```diff
@@ -0,0 +1,98 @@
+from openerp import tools
+from openerp.osv import fields, orm
+
+import logging
+
+logger = logging.getLogger(__name__)
+
+SLA_STATES = [('5', 'Failed'), ('4', 'Will Fail'), ('3', 'Warning'),
+              ('2', 'Watching'), ('1', 'Achieved')]
+
+
+class report_sla(orm.Model):
```

Comment:
Reports are usually use Wizards (TransientModel). Is there a rationale to prefer a regular Model?